SEC/Debt/TradedShortTermDebt.rdf

```diff
@@ -97,12 +99,19 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">bankers&apos; acceptance</rdfs:label>
-		<skos:definition xml:lang="en">a short-term debt instrument issued by a company that is guaranteed by a commercial bank</skos:definition>
+		<skos:definition xml:lang="en">short-term debt instrument issued by a company that is guaranteed by a commercial bank</skos:definition>
```

Comment:
ambiguous whether it's the company or the instrument that's guaranteed.
More importantly I believe the bank not only guarantees the payment but is actually responsible for making the payment